lib/collection/property-base.js

```diff
@@ -91,6 +91,15 @@ _.extend(PropertyBase.prototype, /** @lends PropertyBase.prototype */ {
             accumulator[key] = _.cloneElement(value);
             return accumulator;
         }, {});
+    },
+
+    /**
+     * Returns the meta keys associated with the property
+     *
+     * @returns {*}
+     */
+    meta: function () {
+        return arguments.length ? _.pick(this._, Array.prototype.slice.apply(arguments)) : _.cloneDeep(this._);
```

Comment:
It the _.pick will not clone deep. Also I think lodash handles arguments variable directly.